src/controllers/userController.ts

```diff
@@ -0,0 +1,69 @@
+import { prisma } from "../model/prisma";
+import { Body, Set } from "../utils/utils";
+
+export const user = {
+  getById: async (userId: number, set: Set) => {
+    const user = await prisma.user.findUnique({
+      where: { id: userId },
+    });
+
+    if (!user) {
+      set.status = 400;
+      return "User not found";
+    }
+
+    return user;
+  },
+
+  getAll: () => prisma.user.findMany(),
+
+  post: async (body: Body, set: Set) => {
+    if (!body.email) {
+      set.status = 400;
+      return "Email is required";
+    }
+
+    const user = await prisma.user.create({
```

Comment:
Please wrap it with try catch and throw an error if something went wrong with the DB